### PR TITLE
remove tabs in some docs

### PR DESCRIPTION
**Pull request by rosemarymarano**
@gilbode expressed a degree of displeasure with our (over) use of tabs. I removed them from some docs and am interested in your feedback about whether it's an improvement or not. Also, please check your docs to make sure that the text and procedural flow was not incorrectly interrupted. Thank you.

### CODE REVIEW

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 1]
```diff
@@ -28,12 +28,12 @@ You must have the following software installed:
...
  * `README.md` which provides the layout of the respective distribution
 
-Set up a private registry using the following instructions, depending on your distribution.
-{{< tabs tabTotal="2" >}}
-{{< tab tabName="LiteDistribution" >}}
-<br>
+Set up a private registry using the following instructions, depending on your distribution:
+- [Lite distribution](#lite-distribution)
+- [Full distribution](#full-distribution)
+
```
Comment (MelwynPaul) [inline content/en/docs/setup/private-registry/private-registry-full-distribution.md:34]: @rosemarymarano, only for this topic, I think we should retain the tabs.
Reason: Combination of > Larger amount of info + users would use only type of distribution. 
A correct usage of tabs.
_____________
If we do plan to go ahead and remove the tabs, I am okay with that too.
However, I do not like the new 'Lite and full distributions' section, where 'step 3' is added as a common step after the first 2 steps of 'Lite distribution' and 'Full distribution'. 
Especially, the 'Lite and full distributions' section starting with step number '3'.
Looks a bit odd.

**Review by MelwynPaul — COMMENTED** [review]
I added a few comments.
I am happy that some of these tabs are removed - especially, the ones that have less content.
Thank you. 👍

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md (hunk 1 above)
Comment (rosemarymarano) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md:34]: @MelwynPaul I agree with your assessment about NOT removing the tabs; I've restored them.

File: content/en/docs/setup/install/multicluster.md [hunk 2]
```diff
@@ -74,7 +74,12 @@ Follow these preregistration setup steps.
    cluster will need the managed cluster's CA certificate to make an `https` connection.
    - Depending on whether the Verrazzano installation on the managed cluster uses
      self-signed certificates, LetsEncrypt staging certificates, or certificates signed by a well-known
-     certificate authority, choose the appropriate instructions.
+     certificate authority, choose the appropriate instructions:
+
+     - [Well-known CA](#well-known-ca)
+     - [Self-signed certificates](#self-signed-certificates)
+     - [LetsEncrypt staging certificates](#letsencrypt-staging-certificates)
+
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply content/en/docs/setup/install/multicluster.md:82]: @MelwynPaul Here I disagree with your suggestion; here's why. I don't think that we should make the users read all the content of three security choices: self-signed certificates, LetsEncrypt staging certificates, or certificates signed by a well-known certificate authority, before seeing the next bulleted item, "If you are unsure what type of certificates are used, use the following instructions." If they don't already have it, then they need **that** information before they read the certificate-specific section that pertains to them.

File: content/en/docs/setup/install/multicluster.md [hunk 3]
```diff
@@ -100,133 +105,121 @@ Follow these preregistration setup steps.
...
+       # After the command succeeds, you may delete the managed1.yaml file
+       $ rm managed1.yaml
+       ```
 
 1. Use the following instructions to obtain the Kubernetes API server address for the admin cluster. This address must
    be accessible from the managed cluster.
+     - [Most Kubernetes Clusters](#most-kubernetes-clusters)
+     - [Kind Clusters](#kind-clusters)
+
+     #### Most Kubernetes Clusters
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply content/en/docs/setup/install/multicluster.md:179]: @MelwynPaul Yep, I agree with you here; I'll make that change.

File: content/en/docs/setup/install/multicluster.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MelwynPaul) [reply content/en/docs/setup/install/multicluster.md:82]: @rosemarymarano 
Awesome. Yes, I read the section and agree to your update here. 👍 
Thank you.

**Review by MelwynPaul — APPROVED** [review]
Thank you, @rosemarymarano . 
Looks good.
Approved.

**Pull request merged**